css/index.css

```diff
@@ -1 +1,583 @@
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+/* ol, ul {
+	list-style: none;
+} */
+
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* DESKTOP VIEW CSS */
+body {
+	display: flex;
+	justify-content: center;
+}
+
+.container-div {
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	background: white;
+}
+
+.header-bar {
+	width: 85%;
+	height: 96px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.nav {
+	width: 68%;
+	height: 96px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.nav-button {
+	position: relative;
+	top: 40px;
+	text-decoration: none;
+}
+
+.nav-text {
+    color: rgb(186,186,186);
+    font-size: 1rem;
+    font-weight: bold;
+    
+}
+
+#logo-holder {
+	float: right;
+	height: 96px;
+	width: 20%;
+}
+
+.logo {
+	position: relative;
+	top: 30px;
+}
+
+#under-header{
+	height: 495px;
+	width: 85%;
+	display: flex;
+    justify-content: space-between;
+    align-items: center;
+	flex-wrap: wrap;
+}
+
+#innovation-on-demand {
+    width: 34%;
+    height: auto;
+    left: 15%;
+	text-align: center;
+	font-family: 'Bangers', cursive;
+	font-size: 4.625rem;
+	position: relative;
+}
+
+.get-started-button {
+	width: 60%;
+	height: 35px;
+	border: 1px solid black; 
+	text-align: center;
+	text-decoration: none;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 0.75rem;
+	position: relative;
+	top: -20px;
+	background: white;
+}
+
+#get-started-text {
+    font-size: 1rem;
+}
+
+#image-1 {
+    width: 380px;
+    width: 380px;
+    position: relative;
+    top: -4%;
+    left: 20px;
+}
+
+.line-break {
+	border: 1px solid black;
+	width: 85%;
+	height: 1px;
+}
+
+.main-section {
+	height: 175px;
+	width: 85%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.main-section2 {
+	height: 208px;
+	width: 85%;
+	display: flex;
+	justify-content: space-between;
+    align-items: center;
+}
+
+.title-text {
+	font-size: 1.125rem;
+}
+
+.title-text2 {
+	font-size: 1.125rem;
```

Comment:
By setting font-size to 62.5%, you can get rid of the extra decimals you have here.